src/cursor/cur_stat.c

```diff
@@ -745,7 +767,8 @@ __wt_curstat_open(WT_SESSION_IMPL *session, const char *uri, WT_CURSOR *other, c
     if (0) {
 config_err:
         WT_ERR_MSG(session, EINVAL,
-          "cursor's statistics configuration doesn't match the database statistics configuration");
+          "cursor's statistics configuration doesn't match the database statistics "
+          "configuration");
```

Comment:
Did you intentionally break this into two strings?

Reply:
This was from clang format. I did not change anything there myself.

Reply:
I recombined them and will see if it does it again. The normal combined line is 99 characters so should fit.